src/Tools/GetDocumentInsider/src/Commands/GetDocumentCommandWorker.cs

```diff
@@ -206,18 +209,39 @@ private bool GetDocuments(IServiceProvider services)
         var getDocumentsMethod = GetMethod(
             GetDocumentsMethodName,
             serviceType,
-            GetDocumentsParameterTypes,
-            GetDocumentsReturnType);
+            _getDocumentsParameterTypes,
+            _getDocumentsReturnType);
         if (getDocumentsMethod == null)
         {
             return false;
         }
 
+        var generateWithVersionMethod = serviceType.GetMethod(
+            GenerateMethodName,
+            _generateWithVersionMethodParameterTypes);
+
+        if (generateWithVersionMethod is not null)
+        {
+            if (generateWithVersionMethod.IsStatic)
+            {
+                _reporter.WriteWarning(Resources.FormatMethodIsStatic(GenerateMethodName, serviceType));
```

Comment:
Is this using the original resource string?  Would it be helpful to say that we're going to disregard this member and use the versionless one?

Reply:
Yeah, I ended up addressing this by adding additional info output right before we invoke the method about which overload is being used.